links/ancient-masters.md

```diff
@@ -2,25 +2,25 @@
-* [A list of the Masters](http://jan.ucc.nau.edu/~wew/fencing/masters.html) - by Master Gwylym ab Owain Tatershal
-* [Illustrazioni d'Agrippa](http://jan.ucc.nau.edu/~wew/fencing/agrippa/agrippa_illus.html) - by Camillo Agrippa (1568)
-* [The Four Dialogues of True Skill](http://jan.ucc.nau.edu/~wew/fencing/caranza.html) - by Jeronimo de Caranza (1569)
-* [Dell'arte di scrimia](http://jan.ucc.nau.edu/~wew/fencing/agrippa/agrippa_illus.html) - by Giovannia Dell'Agocchie (1572)
-* [Traite Contenant les Secrets du Premier Livre sur l'Espee Seule etc](http://jan.ucc.nau.edu/~wew/fencing/didier.htm) - by Sainct Didier (1573)
-* [New Kunstliches Fechtbuch](http://fs.spinfo.uni-koeln.de/~dieke/Sutor/) - by Jakob Sutor (1617)
-* [The Teachings of Marozzo](http://jan.ucc.nau.edu/~wew/other/gr/)
-* [Terminology Table from Rapier-1 list](http://www.sonic.net/~willena/TermChart3_02.html)
-* [The Private Schoole of Defense](http://www.thehaca.com/Manuals/FullPSoDtext.htm) - by George Hale (1614)
-* [Pallas Armata - The Gentleman's Armory - Book One, Part One](http://www.thehaca.com/pallas.htm) - (1639)
-* [Pallas Armata - The Gentleman's Armory - Book One, Part Two](http://www.thehaca.com/pallas2.htm) - 1639(
-* [Comprehension of Destreza](http://www.thehaca.com/destreza.htm) - by Alvaro Guerra de la Vega (1681)
-* [Introduction to Frederico Ghisliero's Rapier Text](http://www.thehaca.com/essays/Regole.htm] - by Frederico Ghisliero (1587)
-* [Schoole of the Noble and Worthy Science of Defence](http://www.thehaca.com/swetnam.htm) - by Joeseph Swetnam (1617)
-* [Images from Kunst des Fechten](http://www.thehaca.com/pdf/JoachimMeyer.htm) - by Joachim Meyer (1570)
+* [Paradoxes of Defense](http://www.pbm.com/~lindahl/paradoxes.html){:target="_blank"} - by George Silver (1599)
+* [Brief Instructions](http://www.pbm.com/~lindahl/brief.html){:target="_blank"} - by George Silver (1599)
+* [His Practise, The First Book](http://www.cs.unc.edu/~hudson/saviolo/){:target="_blank"} - by Vincentio Saviolo (1595)
+* [His True Arte of Defence](http://www.cs.unc.edu/~hudson/digrassi/){:target="_blank"} - by Giacomo Di Grassi (1594)
+* [Traite D'Escrime](http://jan.ucc.nau.edu/~wew/fencing/lovino.html){:target="_blank"} - by Gio. Antonio Lovino (1580)
+* [Weapon Seizures](http://jan.ucc.nau.edu/~wew/fencing/seizures.html){:target="_blank"} - by Achille Marozzo (1536)
+* [The Book of the Courtier](http://darkwing.uoregon.edu/~rbear/courtier/courtier.html){:target="_blank"} - Trans. by Sir Thomas Hoby (1561)
+* [A list of the Masters](http://jan.ucc.nau.edu/~wew/fencing/masters.html){:target="_blank"} - by Master Gwylym ab Owain Tatershal
+* [Illustrazioni d'Agrippa](http://jan.ucc.nau.edu/~wew/fencing/agrippa/agrippa_illus.html){:target="_blank"} - by Camillo Agrippa (1568)
+* [The Four Dialogues of True Skill](http://jan.ucc.nau.edu/~wew/fencing/caranza.html){:target="_blank"} - by Jeronimo de Caranza (1569)
```

Comment:
I would prefer links to not create a new window.  All browsers have the ability to open links in new windows at the user's request.   Forcing new window on links breaks the back-forward model that has been a mainstay of browsing for nearly 30 years.